src/components/HomeSection.test.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import Button from '@material-ui/core/Button';
+import HomeSection from './HomeSection';
+
+describe('HomeSection', () => {
+  test('renders the given title', () => {
+    render(
+      <HomeSection
+        title="Topology"
+        description="Math course"
+        buttons={[]}
+        pic="manifold"
+      />
+    );
+    screen.debug();
+    expect(screen.getByText('Topology')).toBeInTheDocument();
+  });
+
+  test('renders given description', () => {
+    render(
+      <HomeSection
+        title="Topology"
+        description="Math course"
+        buttons={[]}
+        pic="manifold"
+      />
+    );
+    expect(screen.getByText('Math course')).toBeInTheDocument();
+  });
+
+  test('renders no buttons by default', () => {
+    render(
+      <HomeSection
+        title="Topology"
+        description="Math course"
+        buttons={[]}
```

Comment:
Delete this line as this test case is testing the default..

Reply:
delete line 37?

Reply:
Of course. Otherwise how else are you supposed to test the _default_ `buttons` value?

Reply:
alright - just making sure.